### PR TITLE
finalized cronjob code

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial <kialj876@gmail.com>

*Issue #:* /bcgov/entity#9834

*Description of changes:*
- added new config options for Bob
- added legacy pull from table instead of csv
- minor updates

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: cronjobs/search-tester/run_search_tester.py [hunk 1]
```diff
@@ -64,6 +65,65 @@ def add_api_results(search: TestSearch, results: List[dict]):
...
+                'index': row[row_keys[6]]
+            })
+    return batch_searches
+
+
+def get_batch_searches_csv(app: Flask, batch_searches: dict) -> dict:
+    """Get batches from csv."""
+    # read csv + prep for search tests
+    filename = app.config['FILE_NAME']
+    with open(f'csvs/{filename}', 'r') as csvfile:
```
Comment (jordiwes) [inline cronjobs/search-tester/run_search_tester.py:106]: Does this need a try/catch?

**Review by jordiwes — APPROVED** [review]
One question, otherwise looks good

**Review by doug-lovett — APPROVED** [review]
LGTM.

File: cronjobs/search-tester/run_search_tester.py (hunk 1 above)
Comment (kialj876) [reply cronjobs/search-tester/run_search_tester.py:106]: The fn is called within a try catch so I think it's okay. It's a temp job so I don't think we need to worry about it too much

**Pull request merged**